### DISCUSSION TITLE
Is it possible to add "Add formatter" similar to "Add filter"

### DISCUSSION THREAD

**Discussion by Elamurugan-Nallathambi** [sevdokimov/log-viewer#121] [Ideas]
What:

- Include a pipe option (Similar to add filter) to format message or date or any part of the event message to particular way

Why:

- If suppose the logs contain url encoded or / escaped lot of strings, the log viewer wants to see them without url encoded, so including a formatter to remove anything thats \/ to / can help in this situation.

**Comment by sevdokimov** [comment]
This should help: https://github.com/sevdokimov/log-viewer/issues/24
